docs/user/software/statelessness.md

```diff
@@ -0,0 +1,97 @@
+
+- User configurations _Highest priority_
+- System configurations
+- OS provided configurations, AKA default configurations _Lowest priority_
+
+### User configurations
+
+The user configurations can be distinct for each user profile on a machine. These configurations are stored in the `XDG_CONFIG_HOME` directory: `~/.config`, while some applications insist on placing configuration files straight into the home directory. For example the `fish` shell uses the `~/.config/fish` directory to store user configurations, while the `bash` shell uses the `~/.basrc`configuration file straight from the home directory.
+
+_The package manager will never touch User configurations._
+
+### System configurations
+
+The system configurations are used by all users on a machine. These configurations are stored in the `/etc` directory, for example, the `fish` shell uses the `/etc/fish` directory.
+
+_The package manager will never touch System configurations._
+
+### OS provided configurations
+
+The OS provided configurations are used by all users on a machine. These configurations are stored in the `/usr/share/defaults` directory. For example, the `sway` window manager uses `/usr/share/defaults/sway`
+
+_The package manager_ **will** _change OS provided configurations._
+
+## Using statlessness
+
+Using statelessness, you can modify configurations knowing you always have the option to return to the Solus defaults. Additionally, your User configurations can contain the minimal set of desired changes and use default configuration for the rest.
+
+To demonstrate this in practice, we will use the `sway` window manager:
+
+Create a directory for your User configuration, then copy the default configuration file provided by Solus:
+
+```bash
+mkdir ~/.config/sway
+cp /usr/share/defaults/sway/config ~/.config/sway/config
+```
+
+:::tip
+
+You could also place this configuration file in `/etc/sway`, where the configuration would apply to all users.
+
+:::
+
+Now you can edit your configuration file:
+
+```bash
+nano ~/.config/sway/config
+```
+
+To return to Solus defaults, remove your configuration file:
+
+```bash
+rm ~/.config/sway/config
+```
+
+If you think you have somehow changed the Solus provided defaults, re-install `sway` to overwrite any configuration in `/usr/share/defaults/sway`:
+
+```bash
+sudo eopkg install --reinstall sway
+```
+
+You can see additional examples on the [Web Development page](/docs/user/software/development/web.md).
+
+## Common default configuration locations
+
+This is a list of common default configurations users may wish to copy and change.
+
+### SSH
+
+`/usr/share/defaults/etc/ssh/`
+
+### TODO: Others?
+
+- pam, ssl
```

Comment:
samba, acupsd

Found this in /etc/, used by the APC UPS daemon (not installed by default, in the repos)
```
 ❯ head /etc/changeme                                                                                                                         [12:43:38]
#!/bin/sh
#
# This shell script if placed in /etc/apcupsd
# will be called by /etc/apcupsd/apccontrol when apcupsd
# detects that the battery should be replaced.
# We send an email message to root to notify him.
```

Reply:
In my mind, this list is for things that a typical user *might* change. I don't want to make an exhaustive list of all the configs we ship. 

For pam and ssl, I'm skeptical we should point users at these configs without some sort of big flashing danger sign